charts/hail-search/templates/_helpers.tpl

```diff
@@ -1,25 +1,13 @@
-{{/*
-Expand the name of the chart.
-*/}}
-{{- define "hail-search.name" -}}
```

Comment:
I got rid of `nameOverride` and `fullNameOverride` and `name` just to simplify the logic.  I think it makes sense to allow external users to override the 'release' but the applications should still be named what they're named.

Reply:
Yeah, I'm fine with this -- these tpls that get generated with the helm create command are pretty broad, intending to cover pretty much any use case, so narrowing this a bit seems fine.